cmd/amp/account.go

```diff
@@ -0,0 +1,116 @@
+package main
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var (
+	// Interactive
+	loginCmd = &cobra.Command{
+		Use:   "login",
+		Short: "log in to amp",
+	}
+
+	accountCmd = &cobra.Command{
+		Use:   "account",
+		Short: "account operations",
+	}
+	// The rest of the commands are subcommands
+
+	// Interactive
+	signUpCmd = &cobra.Command{
+		Use:   "signup",
+		Short: "Create a new account and login",
+	}
+
+	// Interactive
+	verifyCmd = &cobra.Command{
+		Use:   "verify [CODE]",
+		Short: "verify email using code",
+	}
+
+	assumeRoleCmd = &cobra.Command{
+		Use:   "assume [ORGANIZATION] [TEAM]",
```

Comment:
Only need to switch organization for now. If you're on multiple teams with access to a resource, your grant will be the highest level of access available to you.